src/Domain/Fixture/Helper/DrupalSettingsHelper.php

```diff
@@ -79,12 +86,32 @@ private function ensureCiSettingsFile(): void {
   private function ensureLocalSettingsFile(): void {
     $path = $this->fixture->getPath(self::LOCAL_SETTINGS_PATH);
 
-    $data = PHP_EOL;
+    $data = NULL;
+    if ($this->filesystem->exists($path)) {
+      $data .= PHP_EOL;
+    }
+    else {
+      $data .= '<?php' . PHP_EOL . PHP_EOL;
+    }
```

Comment:
I don't understand your suggestion. This function needs to append to a file regardless of the conditional, it can never just return early.

The conditional exists because the behavior is different depending on whether the settings file already exists. If it exists, we append a newline and the settings. If it doesn't exist, we "append" (create, really) a `<?php` tag and the settings.

Reply:
Oh! I altogether overlooked line 83. That makes sense then. Move along. 😄 

```suggestion
    $data = NULL;
    if ($this->filesystem->exists($path)) {
      $data .= PHP_EOL;
    }
    else {
      $data .= '<?php' . PHP_EOL . PHP_EOL;
    }
```